migrations/versions/180_remove_user_locked_flag.py

```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 180_remove_user_locked_flag
+Revises: 170_add_login_tracking
+Create Date: 2015-07-03 16:07:45.172143
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '180_remove_user_locked_flag'
+down_revision = '170_add_login_tracking'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('users', 'locked')
```

Comment:
Do we need the `active` column for anything at all?  If not then we could get rid of that at the same time maybe.